src/bootstrap/Cargo.toml

```diff
@@ -39,7 +39,8 @@ cc = "1.0.69"
 libc = "0.2"
 hex = "0.4"
 object = { version = "0.29.0", default-features = false, features = ["archive", "coff", "read_core", "unaligned"] }
-serde = { version = "1.0.8", features = ["derive"] }
+serde = "1.0.137"
+serde_derive = "1.0.137"
```

Comment:
Without comment, this will be eventually reverted back.

Reply:
Right